server/src/inmantals/server.py

```diff
@@ -102,7 +105,7 @@ def __init__(self, instream: BaseIOStream, outstream: BaseIOStream, address):
 
     async def initialize(self, rootPath, rootUri, capabilities: Dict[str, object], **kwargs):  # noqa: N803
         logger.debug("Init: " + json.dumps(kwargs))
-
+        Config.set("compiler", "cache", "False")
```

Comment:
I don't think we want this in general, just for testing. So you need to set the corresponding env var in the Jenkinsfile.